webapp/publisher/snaps/views.py

```diff
@@ -495,14 +496,15 @@ def get_release_history(snap_name):
 
 
 @publisher_snaps.route("/account/snaps/<snap_name>/release", methods=["POST"])
```

Comment:
Do we need redirect for POST? I don't think anyone would have that bookmarked or linked, and if we update internal URL it should be fine.

Of course, assuming we update our links to `/release` in HTML and API/POST URL in JavaScript.